src/Controls/tests/UITests/Tests/Issues/Issue20723.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue20723 : _IssuesUITest
+	{
+		public override string Issue => "[iOS] PullToRefresh activity indicator indicator hidden behind header in CollectionView";
+
+		public Issue20723(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		public async Task IndicatorShouldNotBeCoveredByHeader()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android, TestDevice.Mac, TestDevice.Windows });
+			_ = App.WaitForElement("refreshView");
+
+			App.ScrollUp("refreshView", swipePercentage: 1);
+			await Task.Delay(500);
+
+			//The test passes if the activity indicator is above the header
+			VerifyScreenshot();
```

Comment:
`ValidateScreenshot ` will compare the screenshots pixel by pixel. Taking into account the animation of the position of the loading indicator as well as the rotation of the indicator itself, I would say that it would be better to create a device test to validate that it is within a range as well as the size.

Reply:
You're right this test will probably fail in most of the cases. However, I'm not sure if the device test will be useful in this case. Maybe we should just not write a test for this issue?

Reply:
Yeah, i think in this case have sense a tests checking Frames etc. Take a look to the `GetRect `extension method used in UITests. Let me know if need help with something.

Reply:
![image](https://github.com/dotnet/maui/assets/6755973/0d38c718-e23a-4042-948a-ca13b2bbed7a)
The snapshot, with the loading indicator is not a valid option.

Also, can bind the status to a Label, and check in the test the text value Example: "Loading", "Not Loading".

Reply:
I'm not sure how to get frames of this indicator in any other way than using XPath
<img width="1670" alt="Screenshot 2024-04-16 at 19 32 26" src="https://github.com/dotnet/maui/assets/42434498/b9aa8bc2-e229-4350-b817-7169c2e019e8">